company-profiles/fraudio.md

```diff
@@ -0,0 +1,37 @@
+# Fraudio
+
+## Company blurb
+
+Our startup https://fraudio.com is bringing the latest AI technology to the finance and payments sector and is looking to disrupt it. 
+We use cutting-edge technology and research-driven techniques to create tools that productionize AI for our clients, to combat various illicit activites 
+such as transaction fraud, merchant fraud and money laundering.
```

Comment:
There is an odd line break here.

Reply:
Fair enough! I removed the unnecessary line breaks.